### ISSUE TITLE
Keyboard navigation doesn't work in the Min-Vid player for Vimeo videos

### ISSUE THREAD

**Issue by SoftVision-CiprianMuresan** [meandavejustice/min-vid#384]
[Affected versions]:
- Firefox 45.0 and up
- Min-Vid 0.3.0-dev

[Affected Platforms]:
- All Windows
- All Mac
- All Linux

[Prerequisites]:
- Latest Min-Vid add-on version (0.3.0-dev) was previously installed on a clean profile.

[Steps to reproduce]:
1. Open the browser with the profile from prerequisites and navigate to Vimeo.
2. Send a video to the mini player and choose to play it.
3. Press either arrows(Up/Down/Left/Right) from the keyboard or the Space bar and observe the behaviour.

[Expected result]:
- Left/Right arrows fast forward or backward the video, Up/Down arrows turn the volume Up or Down and the Space bar Pauses/Plays the video.

[Actual result]:
- Nothing happens.

[Additional Notes]:
- Keyboard navigation works on the website player.
- Keyboard navigation works with videos sent from YouTube.
- Attached a screen recording of the issue https://goo.gl/ZZgdlm.

**Comment by meandavejustice** [comment]
closing for #67
